docs/js/doc.js

```diff
@@ -16,9 +16,16 @@
       apiKey: '6be0576ff61c053d5f9a3225e2a90f76',
       indexName: 'instant_search'
     };
+    var initialSearchState = {
+      hitsPerPage: 6,
+      disjunctiveFacets: ['brand'],
+      disjunctiveFacetsRefinements: {
+        brand:['Samsung'] 
+      }
+    };
     var codeSnippets = q('.code-sample-snippet:not(.start):not(.config):not(.ignore)');
     var configSnippet = "var search = instantsearch({appId: '$appId', apiKey: '$apiKey'," +
-      "indexName: '$indexName', searchParameters: {hitsPerPage: 6}});\n"
+      "indexName: '$indexName', searchParameters: " + JSON.stringify(initialSearchState) + "});\n"
```

Comment:
Are we sure this is what we want to show to the user? I thought we just wanted to show the hitsPerPage